src/devices/shinasystem.ts

```diff
@@ -1119,4 +1119,61 @@ export const definitions: DefinitionWithExtend[] = [
             await endpoint.read("ssIasZone", ["currentZoneSensitivityLevel"]);
         },
     },
+    {
+        zigbeeModel: ["TQM-300Z"],
+        model: "TQM-300ZB",
+        vendor: "ShinaSystem",
+        ota: true,
+        description: "SiHAS Round Temperature/Humidity Sensor",
+        extend: [m.temperature({reporting: {min: 30, max: 300, change: 30}}), m.humidity({reporting: {min: 30, max: 3600, change: 60}}), m.battery()],
```

Comment:
```suggestion
        extend: [m.temperature(), m.humidity(), m.battery()],
```

Could you use the default reporting intervals?

Reply:
Ok. thanks.